file-formats/sprv/README.md

```diff
@@ -2,4 +2,4 @@
 
 File | Cardinality | Definition | Schema | Example
 :--- | :--- | :--- | :--- | :---
-`<name>.sprv.json` | 1 | [`zif_aff_sprv_v1.intf.abap`](./type/zif_aff_sprv_v1.intf.abap) | [`sprv-v1.json`](./sprv-v1.json) | [`srt_test_sprv.sprv.json`](./examples/srt_test_sprv.sprv.json)
+`<name>.sprv.json` | 1 | [`zif_aff_sprv_v1.intf.abap`](./type/zif_aff_sprv_v1.intf.abap) | [`sprv-v1.json`](./sprv-v1.json) | [`srt_test_sprv2.sprv.json`](./examples/srt_test_sprv2.sprv.json)
```

Comment:
```suggestion
`<name>.sprv.json` | 1 | [`zif_aff_sprv_v1.intf.abap`](./type/zif_aff_sprv_v1.intf.abap) | [`sprv-v1.json`](./sprv-v1.json) | [`srt_test_sprv.sprv.json`](./examples/srt_test_sprv.sprv.json)
```
You might want to revert the filename of the example. Please dont forget to rename the actual file back to its original name `srt_test_sprv.sprv.json`.

Reply:
It would be good to leave it like this as SRT_TEST_SPRV2 is a transported test object that also has a valid implementing class connected while SRT_TEST_SPRV was a more or less emptier test object with no valid proxy test objects behind it. So if it is no problem it can stay from my pov.

Reply:
If so, then also adopt the file name of the example 😄

Reply:
```suggestion
`<name>.sprv.json` | 1 | [`zif_aff_sprv_v1.intf.abap`](./type/zif_aff_sprv_v1.intf.abap) | [`sprv-v1.json`](./sprv-v1.json) | [`z_aff_example_sprv.sprv.json`](./examples/z_aff_example_sprv.sprv.json)
```
Like this? Changed to the new naming pattern